### DISCUSSION TITLE
Open repo in Codespaces

### DISCUSSION THREAD

**Discussion by viceice** [renovatebot/renovate#14682] [Request Help]
> Works for me on WSL v2 and Windows 11. Don't have access to codespaces, so can't test there

I thought codespaces was open to everyone now?

_Originally posted by @JamieMagee in https://github.com/renovatebot/renovate/issues/14652#issuecomment-1068698783_

**Comment by viceice** [comment]
![image](https://user-images.githubusercontent.com/1798109/158559628-27233dab-0fc4-4105-ac27-a1d33e4c0a82.png)
🤷‍♂️

**Reply by rarkins** [reply]
Works for me, but I remember I was on some waiting list about codespaces for a while before being accepted long ago

**Reply by rarkins** [reply]
```
=============================== Coverage summary ===============================
Statements   : 100% ( 25996/25996 )
Branches     : 98.15% ( 8533/8693 )
Functions    : 100% ( 3106/3106 )
Lines        : 100% ( 25559/25559 )
================================================================================

Test Suites: 1 skipped, 422 passed, 422 of 423 total
Tests:       8 skipped, 8269 passed, 8277 total
Snapshots:   2014 passed, 2014 total
Time:        459.826 s
Ran all test suites.
Done in 627.73s.
```

**Reply by rarkins** [reply]
https://github.com/features/codespaces/signup

**Reply by viceice** [reply]
![Screenshot_20220316-155303.png](https://user-images.githubusercontent.com/1798109/158618855-3b515434-8dff-457f-8329-942ec6ce837d.png)

😕
